_includes/layout/edition-callout.html

```diff
@@ -22,6 +22,14 @@
   </span>
 </div>
 {% endif %}
+{% if page.pwa_only == true %}
+<div class="break">
+  <span class="edition-Label pwa-only">{{site.data.var.pwa}} only.
+    <a class="spectrum-Link" href="https://docs.magento.com/user-guide/getting-started.html#product-editions">
```

Comment:
Consider getting rid of the link. I doubt Merchdocs/ExL would have PWA features.